segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -937,10 +941,9 @@ private Void handleException(long requestId, String segment, long offset, String
             log.info(requestId, "Closing connection {} while performing {} due to {}.",
                      connection, operation, u.getMessage());
             connection.close();
-        } else if (u instanceof AuthenticationException) {
-            log.warn(requestId, "Authentication error during '{}'.", operation);
+        } else if (u instanceof TokenException) {
```

Comment:
Same question here about the exception type change.

Reply:
Please refer to the response for a similar question earlier: https://github.com/pravega/pravega/pull/3887#discussion_r294134259